app/src/main/res/layout/activity_widget_authentication.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You aren't inflating this file, so it can be completely removed.

Reply:
OK